src/hotel_reservations/data_processing/workflows/data_processing_task.py

```diff
@@ -14,22 +16,36 @@
 
 
 def preprocessing():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--simulate_drift",
+        action="store",
+        default="False",
+        type=str,
+        required=True,
+    )
```

Comment:
I'm aware of the security risk of using the eval() function, however, it will not be called on user input and therefore does not pose a big security risk